### PR TITLE
feat: add helm chart

### PR DESCRIPTION
**Pull request by oslo254804746**

### CODE REVIEW

**Comment by boojack** [comment]
@h3n4l PTAL

File: deploy/helm/README-zh.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+指定`namespace`示例
+```shell
+helm install memos -n app .
+```
+## 自定义存储
+修改`values.yaml`
+```yaml
+persistence:
+  ## 如果enabled 为False, 会使用emptyDir, Pod 重建后数据可能丢失, 建议使用持久卷
+  ## If enabled is False, emptyDir will be used, data may be lost after Pod rebuild, it is recommended to use persistent volume
```
Comment (h3n4l) [inline deploy/helm/README-zh.md:15]:
```suggestion
```

File: deploy/helm/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+```
+use custom `namespace` example
+```shell
+helm install memos -n app .
+```
+## Persistence
+Edit `values.yaml`
+```yaml
+persistence:
+  ## 如果enabled 为False, 会使用emptyDir, Pod 重建后数据可能丢失, 建议使用持久卷
```
Comment (h3n4l) [inline deploy/helm/README.md:14]:
```suggestion
```

File: deploy/helm/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  ## If enabled is False, emptyDir will be used, data may be lost after Pod rebuild, it is recommended to use persistent volume
+  enabled: false
+  storageClass: "local"
+  existPersistClaim: ""
+  accessMode: ReadWriteOnce
+  size: 10Gi
+```
+when enabled is `false` use `emptyDir`,
```
Comment (h3n4l) [inline deploy/helm/README.md:22]:
```suggestion
when `enabled` is `false` use `emptyDir`, 
```

File: deploy/helm/values.yaml [hunk 4]
```diff
@@ -0,0 +1,96 @@
+# Default values for memos.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+
+  # repo: docker.io
+  # imageName: neosmemo/memos
```
Comment (h3n4l) [inline deploy/helm/values.yaml:10]:
```suggestion
```

**Review by h3n4l — APPROVED** [review]
Some nits, LGTM otherwise

File: deploy/helm/README.md (hunk 2 above)
Comment (oslo254804746) [reply deploy/helm/README.md]: removed

File: deploy/helm/README.md (hunk 3 above)
Comment (oslo254804746) [reply deploy/helm/README.md]: fixed

File: deploy/helm/values.yaml (hunk 4 above)
Comment (oslo254804746) [reply deploy/helm/values.yaml]: removed

**Review by boojack — APPROVED** [review]
LGTM, thanks for your contribution!

**Pull request merged**